host_core/test/host_core/policy_test.exs

```diff
@@ -1,10 +1,12 @@
 defmodule HostCore.PolicyTest do
-  # Any test suite that relies on things like querying the actor count or the provider
-  # count will need to be _synchronous_ tests so that other tests that rely on that same
-  # information won't get bad/confusing results.
+  # TODO
```

Comment:
Is this a blocker?

Reply:
I don't think so. I think this is just a mismatch between how the original test was designed and how policy checking works in the new supervision tree. I'd like to 🍐 with someone to go over rewriting this test at some point but I don't think it's blocking this PR.

Reply:
Okay, just wanted to make sure one run of this test doesn't interfere with subsequent runs